### DISCUSSION TITLE
Cluster Upgrade instructions are wrong

### DISCUSSION THREAD

**Discussion by javierder** [ant-media/Ant-Media-Server#6477] [Q&A]
Hi,

The Cluster Upgrade instructions https://antmedia.io/upgrade-ant-media-server-on-aws-cluster/ start with a link to t http://image-id.antmedia.io/, to find the ID of the AMI, but that site fails.
The image shown in the blog post shows "ami.antmedia...", but that url just redirects to antmedia home.
I need to upgrade from 2.7 to the latest version, please let me now how to proceed.
Regards,
Javier

**Comment by muratugureminoglu** [comment]
Hi @javierder 

Please email us to support@antmedia.io and we will help you.  We can arrange a meeting and guide you.

I'm looking forward to hearing from you. 

Regards.

**Comment by javierder** [comment]
Hi,

I was able to do the upgrade by finding the 2.9 AMI version in Amazon AWS, but the tutorial should be updated anyway :)

Javier

**Reply by mekya** [reply]
Hi @muratugureminoglu,

Javier mentioned updating the document. Do you think it’s a good idea to also update the tutorial? Could there be an opportunity to improve it?

**Reply by yashtandon113** [reply]
@mekya

Yes, we need to update both the document and the blog post.

We also have a document task on backlog: https://github.com/ant-media/ant-media-documentation/issues/285

We can plan it for the next documentation sprint.

**Reply by mekya** [reply]
Thank you @yashtandon113 for your comment. 

@javierder, thank you for the suggestion, follow-up, and everything. 

Let us know if we can help you further. 

Cheers

**Comment by muratugureminoglu** [comment]
Hi @javierder 

Good to hear that your issue is resolved. By the way, http://image-id.antmedia.io/ is working, there may be short outage. Please let me know if you have further questions. I will be here for you :)

![image](https://github.com/ant-media/Ant-Media-Server/assets/6337605/64854bae-8208-40bf-a79b-79b0108d981a)
